Assets/Scripts/FFSoakingSpirits/FFFire.cs

```diff
@@ -81,6 +100,17 @@ void Update()
         if(fireLevel <= 2)
         {
             fireLevel++; // up to Level2 which maxLevel could be a variable maybe?
```

Comment:
would this not allow it to go to 3? i feel like at the end your firelevel would be bouncing between 2 and 3 going into the if then else each other frame

Reply:
good thing the function is only called if the fire is < level2. that's a whoops that was accidentally avoided.